common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -474,6 +474,8 @@ enum class eChannelScanOperationCode : uint8_t {
 #define CHANNEL_SCAN_INVALID_PARAM -1
 #define SCAN_ALL_CHANNELS 0
 
+#define PARAMETER_NOT_CONFIGURED -1
+
 } // namespace beerocks
 
```

Comment:
The commit message is unreadable:
1. what does **this** commit is for - if it is preparative commit (such as is in this case) what is it meant to enable?
2. we decided **not** to limit the `eTriState` to persistent data only - since run-time configuration can also be "not-configured"

try to be general enough not to be forced to change the commit message due to naming on one hand - but describe the added functionality on the other hand, e.g.:
`add a definition on not-configured for optional configurations and parameters not configured yet`

Reply:
I'm not 100% sure what you meant in your comment, but I tried making the commit message more understandable

Reply:
Understood, changed